  * patches/bts.diff: Point to information on how to report bugs in Debian,
    and not to Launchpad. Also update the version number.
--- a/command-not-found
+++ b/command-not-found
@@ -6,7 +6,7 @@ from __future__ import absolute_import,
 
 
 __version__ = "0.3"
-BUG_REPORT_URL = "https://bugs.launchpad.net/command-not-found/+filebug"
+BUG_REPORT_URL = "http://www.debian.org/Bugs/Reporting"
 
 try:
     import sys
